tests/test_gear_yaml2cfg.py

```diff
@@ -97,3 +117,11 @@ def test_yaml2cfg_test_no_header():
         shallow=False,
         )
     p.unlink()
+
+
+def test_yaml_duplicated_params(default_yaml_files):
+    """Make sure no duplicated parameters are present in a ymal file."""
+    assert default_yaml_files != []
+    # Loop over default yaml files
+    for default_yaml_fpath in default_yaml_files:
+        validate_defaults_yaml(default_yaml_fpath)
```

Comment:
This is not testing anything. I see that if `default_yaml_fpath` is not valid it will raise a `ConfigurationError` - here you need two assert the two routes, one in which this exception is raised and one in which it's not